packages/opentelemetry-propagator-jaeger/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@opentelemetry/propagator-jaeger",
+  "version": "0.18.0",
+  "description": "OpenTelemetry Jaeger propagator provides HTTP header propagation for systems that are using Jaeger HTTP header format.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts' --exclude 'test/index-webpack.ts'",
+    "test:browser": "nyc karma start --single-run",
+    "tdd": "npm run tdd:node",
+    "tdd:node": "npm run test -- --watch-extensions ts --watch",
+    "tdd:browser": "karma start",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile",
+    "version:update": "node ../../scripts/version-update.js",
+    "watch": "tsc -w"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "jaeger"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
8.5.0

Reply:
There seems to be a number of places where `8.0.0` is still specified (e.g. <https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-exporter-jaeger/package.json#L29>), is there some kind of a rule for this? Anyway, fixed!

Reply:
When we started the project we simply started with node `8.0.0` support because that was the same as opencensus, however at some point we started to use `perf_hooks` which is only available starting `8.5.0`. We then choose to up the minimum version for all new packages to `0.8.5` to keep consistency across the core packages (then same logic for contribs).